docs/configuration.md

```diff
@@ -0,0 +1,541 @@
+Config file: `"mutation-level": 'Advanced'`
+
+Stryker supports multiple mutation levels. Each level comes with a specific set of mutations. Each level contains the mutations of the levels below it. By setting the level to `Complete` you will get all possible mutations and the thus the strictest mutation test. This comes at the price of longer runtime as more mutations will be generated and tested. 
+
+The levels are:
+- Basic
+- Standard
+- Advanced
+- Complete
+
+| Mutations| Level| 
+| ------------- | ------------- | 
+| Arithmetic Operators | Basic|
+| Block (not yet implemented) | Basic|
+| Equality Operators | Standard |
+| Boolean Literals | Standard|
+| Assignment statements | Standard |
+| Collection initializer | Standard |
+| Unary Operators | Standard |
+| Update Operators | Standard |
+| String Literals and Constants | Standard |
+| Bitwise Operators | Standard |
+| Linq Methods | Standard |
+| Checked Statements | Standard |
+| Regex | Advanced |
+| Advanced Linq Methods (not yet implemented) | Complete |
+| Advanced Regex (not yet implemented) | Complete |
+
+### `reporter` <`string[]`>
+
+Default: `html, progress`  
+Command line: `[-r|--reporter] "html" -r "json" -r "progress"`  
+Config file: `"reporters": ['html', 'json', 'progress']`
+
+*The reporter option can be used multiple times on the command line*
+
+During a mutation testrun one or more reporters can be enabled. A reporter will produce some kind of output during or after the mutation testrun.
+
+The available reporter options are
+* all (Enable all reporters)
+* [html](./reporters.md#html-reporter)
+* [progress](./reporters.md#progress-reporter)
+* [dashboard](./reporters.md#dashboard-reporter)
+* [cleartext](./reporters.md#cleartext-reporter)
+* [cleartexttree](./reporters.md#cleartext-tree-reporter)
+* [dots](./reporters.md#dots-reporter)
+* [json](./reporters.md#json-reporter)
+
+You can find a description for every reporter in the [reporter docs](./reporters.md)
+
+### `additional-timeout` <`number`>
+
+Default: `5000`  
+Command line: `N/A`  
+Config file: `"additional-timeout": 3000`
+
+Some mutations can create endless loops inside your code. To detect and stop these loops Stryker cancels a unit test run after a set time.
+The formula to calculate the timeout is:
+
+`timeout = initialTestTime + additionalTimeout`
+
+If you have a lot of timeouts you might need to increase the additional timeout. If you have a lot of endless loops causing a long mutation testrun you might want to decrease the additional timeout. Only decrease the additional timeout if you are certain that the mutations are endless loops.
+
+*\* Timeout is in milliseconds.*
+
+### `concurrency` <`number`>
+
+Default: `your number of logical processors / 2`  
+Command line: `[-c|--concurrency] 10`  
+Config file: `"concurrency": 10`
+
+Change the amount of concurrent workers Stryker uses for the mutation testrun. Defaults to using half your logical (virtual) processor count.
```

Comment:
```suggestion
Change the amount of concurrent workers Stryker uses for the mutation testrun. Defaults to using half your logical (virtual) processor count.
Set it to *1* to disable parallel testing. This can prevent false positives when your tests affect each other. Setting this option to 1 will decrease performance drastically.  
```